tests/Integration/SearchTest.php

```diff
@@ -7,12 +7,21 @@
 
 class SearchTest extends TestCase
```

Comment:
Let's default to `final` here.

Reply:
We use phpunit ^4.8, there is an error if we use final class as testClass